<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JUQ15IP" failures="1" tests="3" name="Functional-chrome" time="9.473" errors="0" timestamp="2020-11-19T12:17:34 EST">
  <testcase classname="com.vndly.testCases.TC001_Student_RegistrationForm" name="TC_fill_Student_RegistrationForm" time="0.484"/>
  <testcase classname="com.vndly.testCases.TC001_Student_RegistrationForm" name="validateText" time="0.035"/>
  <testcase classname="com.vndly.testCases.TC001_Student_RegistrationForm" name="unValidedText_Capture_A_Screenshot" time="0.025">
    <failure type="java.lang.AssertionError" message="expected [Thanks for submitting the formA] but found [Thanks for submitting the form]">
      <![CDATA[java.lang.AssertionError: expected [Thanks for submitting the formA] but found [Thanks for submitting the form]
at com.vndly.testCases.TC001_Student_RegistrationForm.unValidedText_Capture_A_Screenshot(TC001_Student_RegistrationForm.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- unValidedText_Capture_A_Screenshot -->
</testsuite> <!-- Functional-chrome -->
